esmvaltool/_recipe.py

```diff
@@ -510,9 +510,9 @@ def _update_fx_settings(settings, variable, config_user):
             drs=config_user['drs'])
 
         # allow both sftlf and sftof
-        if fx_files_dict['sftlf']:
+        if fx_files_dict.get('sftlf'):
```

Comment:
Why is `settings['mask_landsea']['fx_files']` a list instead of a single file? I.e. `settings['mask_landsea']['fx_file']`? It is my impression that models have either an sftof or an sftlf file available, but never both because that would be redundant?

Reply:
No, many models (I think most of them) have both files available.

Reply:
Okay, I will test it with #580 then :+1: